src/Microsoft.PowerShell.MarkdownRender/VT100EscapeSequences.cs

```diff
@@ -295,7 +308,14 @@ public string FormatHeader1(string headerText)
         /// <returns>Formatted Header 2 string.</returns>
         public string FormatHeader2(string headerText)
         {
-            return string.Concat(Esc, options.Header2, headerText, endSequence);
+            if (options.EnableVT100Encoding)
```

Comment:
There's a lot of similar code, perhaps have a helper that does the `EnableVT100Encoding` check and concats or just returns the string?

Reply:
Fixed for header types. For others the code is slightly different for each so kept it as is.